lib/simulator-xcode-6.js

```diff
@@ -100,16 +100,10 @@ class SimulatorXcode6 extends EventEmitter {
    * or, for iOS 7.1, the app name without `.app` (e.g., MobileSafari)
    */
   async getAppDir (id, subDir = 'Data') {
-    if (await this.isFresh()) {
-      log.info('Attempted to get an app path from a fresh simulator ' +
-               'quickly launching the sim to populate its directories');
-      await this.launchAndQuit();
```

Comment:
Simulator restart is just a side effect for the **getAppDir** method.  If we really need to stop and start Simulator for such cases then it should happen in the corresponding methods where these properties are being set.